exasol_integration_test_docker_environment/lib/test_environment/spawn_test_environment_with_docker_db.py

```diff
@@ -67,6 +76,12 @@ def create_network_task(self, attempt: int):
             attempt=attempt,
         )
 
+    def _executor_factory(self, database_info: DatabaseInfo) -> DbOsExecFactory:
```

Comment:
This function is probably better in the spawn database task the the task returns the factory. I suggest, to do this refactoring in a separate PR